src/Microsoft.Health.Fhir.Azure.UnitTests/IntegrationDataStore/AzureBlobIntegrationDataStoreClientTests.cs

```diff
@@ -0,0 +1,334 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Storage.Blob;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Fhir.Azure.IntegrationDataStore;
+using Microsoft.Health.Fhir.Core.Configs;
+using Microsoft.Health.Fhir.Core.Features.Operations;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Azure.UnitTests.IntegrationDataStore
+{
+    public class AzureBlobIntegrationDataStoreClientTests
+    {
+        [Fact(Skip = "Local tests need emulator.")]
```

Comment:
What is the use case for these? It looks like they all need the emulator to run.

Reply:
Currently integration test do not support deploy storage. So I created these tests for local check.